duckduckgo/chrome-zeroclickinfo
# Add Sass and DDG styles

**Reviewer:** @jdorweiler 



## Description:

Following from #59, add Sass, and Grunt tasks to build it.
Also make popup.css into a Sass file to verify the Grunt task works.
The setting up and building process remains unchanged, so no need to modify the README.

## Steps to test this PR:
We want to make sure the styles are exactly the same as before. The only piece affected is the popup:
1. Load any webpage
2. Open popup and verify it looks exactly the same as usual
3. Try different things: enabling/disabling the privacy features, social blocking, and see how the popup changes

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
